### DISCUSSION TITLE
Unable to add files to repository

### DISCUSSION THREAD

**Discussion by madismthn** [community/community#88505] [Repositories]
### Select Topic Area

Bug

### Body

I try to add files to my repository online. But every time I see this message 'Error
Looks like something went wrong!'. Whats wrong? How do I fix this?

**Comment by ghost** [comment]
Hey @madismthn try refreshing the page or clearing browser cache or even try in another browser....

Check whether the file is completely uploaded , if you commit while its still uploading it would show this message.

Check your internet connection is proper or not....

Check whether the file you are updating is proper or not in a very large size....

Better try using git to upload your files to the repository....


Hope this helped...

Thank you

**Comment by supuna97** [comment]
Open your repository on the GitHub browser IDE (press the dot(.) key) and upload the files and commit. It works well without errors.
